main.go

```diff
@@ -28,16 +27,17 @@ func main() {
 		log.Fatalf("Failed to create new db %v", err)
 	}
 
+	s := http.Server{
+		EmbedRootDir: static,
+		DB:           d,
+		Template:     t,
+		Key:          key,
+	}
```

Comment:
yes, but I would leave that _as late as possible_ in the function.
Actually where it was before was good, so you can move it back just before you call `s.SetupMux()`.